cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -195,19 +204,35 @@ public void setFields(KafkaAssemblyOperatorMockTest.Params params) {
         this.kafkaStorage = params.kafkaStorage;
 
         this.resources = params.resources;
-        this.before();
     }
 
-    private Vertx vertx;
     private Kafka cluster;
 
-    public void before() {
-        this.cluster = new KafkaBuilder()
-                .withMetadata(new ObjectMetaBuilder()
-                        .withName(CLUSTER_NAME)
-                        .withNamespace(NAMESPACE)
-                        .withLabels(TestUtils.map("foo", "bar"))
-                        .build())
+    @BeforeAll
+    static void setup() {
```

Comment:
In #2737 I tended to use `before` and `after` which seemed to be used most often as method names. I do not have strong preference, but we should try to use the same everywhere.